app/components/molecules/section-info/section-info.component.tsx

```diff
@@ -0,0 +1,38 @@
+import {FC} from 'react';
+
+import events from './img/events.svg';
+import games from './img/games.svg';
+import ingredients from './img/ingredients.svg';
+import internet from './img/internet.svg';
+import {LeadingIcon, Props, StyledSectionInfo} from './section-info.styles';
+import {Hr} from '../../atoms/hr';
+import {Paragraph} from '../../atoms/paragraph';
+
+export const SectionInfo: FC<Props> = ({children, variant = 'none'}) => {
```

Comment:
why make the default variant `'none'` if there is no `none` variant?, are we reusing this component in another part?

AFAIK those are the variants:
![image](https://user-images.githubusercontent.com/339040/229646414-790923f1-d2f6-4cb4-be33-cc7483d32ebd.png)

Just accept the value as it is, none is undefined